store/index.js

```diff
@@ -1,6 +1,45 @@
 import { defineStore } from 'pinia';
 import axios from 'axios';
 
+const monsterFields = [
+  'slug',
+  'name',
+  'challenge_rating',
+  'type',
+  'size',
+  'hit_points',
+  'document__slug',
+  'document__title',
+];
+
+const monsterOrder = 'slug';
+
+const spellFields = [
+  'slug',
+  'name',
+  'school',
+  'dnd_class',
+  'spell_lists',
```

Comment:
Before we deploy to main, we should check that the live API has also received the update to support the new spell_lists value